### PR TITLE
Github & Heroku config added to ssokenizer.yml

### PR DESCRIPTION
**Pull request by chrisbh4**

### CODE REVIEW

File: etc/ssokenizer.yml [hunk 1]
```diff
@@ -38,6 +38,48 @@ identity_providers:
...
+    client_secret: "$GITHUB_CLIENT_SECRET"
+
+    # where user should be sent with sealed OAuth token after authenticating
+    return_url: "$GITHUB_RETURN_URL"
+
+    # OAuth scopes to request. Separate scopes with spaces to specify multiple
+    # scopes in the same line. This makes pulling them from environment
+    # variables easier.
+    scopes:
+      - "repo,user:email"
```
Comment (btoews) [inline etc/ssokenizer.yml]: Let's use the `GITHUB_SCOPES` environment variable here. I've set this appropriately in the production deployment, but you'll need to set it in your `.envrc` for running this locally.

File: etc/ssokenizer.yml [hunk 2]
```diff
@@ -38,6 +38,48 @@ identity_providers:
...
+    client_secret: "HEROKU_CLIENT_SECRET"
+
+    # where user should be sent with sealed OAuth token after authenticating
+    return_url: "HEROKU_RETURN_URL"
+
+    # OAuth scopes to request. Separate scopes with spaces to specify multiple
+    # scopes in the same line. This makes pulling them from environment
+    # variables easier.
+    scopes:
+      - "repo,user:email"
```
Comment (btoews) [inline etc/ssokenizer.yml]: Same here. For Heroku, it looks like we currently request `identity,read-protected,write` scopes.

File: etc/ssokenizer.yml [hunk 3]
```diff
@@ -38,6 +38,48 @@ identity_providers:
     # variables easier.
     scopes:
       - "$GOOGLE_SCOPES"
+      
+  github:
+    # amazon, bitbucket, facebook, github, gitlab, google, heroku, microsoft,
```
Comment (btoews) [inline etc/ssokenizer.yml]: Let's remove all the comments from the `github`/`heroku` sections. I put these in the `google` section so people looking at the repo would know what they do, but it's redundant to spell that out for each provider.

**Review by btoews — CHANGES_REQUESTED** [review]
I merged `main` into your branch since I'd made some other changes to `ssokenizer.yml`. We're now pulling the port for the `http.address` from the `PORT` environment variable. So, you'll want to set `export PORT=8824` in your `.envrc` for running locally.

**Review by chrisbh4 — COMMENTED** [review]
I pushed up the new changes for the config to use env variables for scoping and also removed the duplicated comments in the Github and Heroku section. I also set the envrc port=8824.

File: etc/ssokenizer.yml [hunk 4]
```diff
@@ -38,6 +38,32 @@ identity_providers:
...
+
+    return_url: "$GITHUB_RETURN_URL"
+
+    scopes:
+      - "$GITHUB_SCOPES"
+
+  heroku:
+    profile: heroku
+
+    client_id: "HEROKU_CLIENT_ID"
```
Comment (btoews) [inline etc/ssokenizer.yml]: This is missing a $, that's why it's not pulling the value from the environment variable

Comment (chrisbh4) [reply etc/ssokenizer.yml]: That makes sense of course it would be a missing syntax smh but I just pushed up the fix and the client_id is now showing

**Review by btoews — APPROVED** [review]
This looks good. I'll try to get the `HEROKU_*` variables set in production in the next few days and then merge this.

**Comment by chrisbh4** [comment]
Sounds good just dm me on slack when the variables are set and I'll merge the PR

**Comment by btoews** [comment]
The new secrets are set in prod and I've deployed this branch.

**Pull request merged**